src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -1222,39 +1222,28 @@ private MergedTransactionCommand ReadCommand(string type, BlittableJsonReaderObj
 
             if (cmd == null)
             {
-                //Todo to decide what to do when can't read command - "If there is an error, using the PeepingTomStream to provide context"
-                throw new Exception("Can't read MergedTransactionCommand for replay");
+                throw new ReplayTransactionsException($"Can't read {type} for replay", peepingTomStream);
             }
             return cmd;
         }
 
-        public void JustForKeepTheCode()
+        public class ReplayTransactionsException : Exception
         {
-            //            switch (type)
-            //            {
-            //                case "BeginTx":
-            //                    disposeTxCtx = _parent.DocumentsStorage.ContextPool.AllocateOperationContext(out txCtx);
-            //                    txCtx.OpenWriteTransaction();
-            //                    break;
-            //                case "Commit":
-            //                    txCtx.Transaction.Commit();
-            //                    disposeTxCtx.Dispose();
-            //                    break;
-            //                case "Rollback":
-            //                    // without commit, rolling back
-            //                    txCtx.Transaction.Dispose();
-            //                    disposeTxCtx.Dispose();
-            //                    break;
-            //                default:
-            //                    item.TryGet(CommandKey, out BlittableJsonReaderObject commandReader);
-            //                    var cmd = ReadCommand(type, commandReader, context);
-            //                    if (cmd == null)
-            //                    {
-            //                        continue;
-            //                    }
-            //                    cmd.Execute(txCtx, _recordingStatus);
-            //                    break;
-            //            }
+            public string Last4Kb { get;}
```

Comment:
This may not be 4KB in size. Better to just say "context" or something like this

Reply:
done